src/common/filament_sensor.c

```diff
@@ -23,20 +23,18 @@
 static volatile fsensor_t state = FS_NOT_INICIALIZED;
 static volatile fsensor_t last_state = FS_NOT_INICIALIZED;
 
-
-typedef enum{
-    M600_on_edge  = 0,
-    M600_on_level = 1,
-    M600_never    = 2
-}
-send_M600_on_t;
+typedef enum {
+    M600_on_edge,
```

Comment:
I know the previous code was kind of redundant but it was also more readable (explicit values)...